variation-commons-core/src/test/java/uk/ac/ebi/eva/commons/core/models/AbstractVariantTest.java

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright 2018 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.ac.ebi.eva.commons.core.models;
+
+import org.hamcrest.CoreMatchers;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import uk.ac.ebi.eva.commons.core.models.pipeline.Variant;
+
+import static org.junit.Assert.assertEquals;
+
+public class AbstractVariantTest {
+
+    @Rule
+    public ExpectedException exception = ExpectedException.none();
+
+    @Test
+    public void testNegativeStartCoordinateInConstructor() throws IllegalArgumentException {
+        exception.expect(IllegalArgumentException.class);
+        exception.expectMessage(CoreMatchers.equalTo("Variant start coordinate cannot be negative"));

```

Comment:
The messages can be extracted in constants in the AbstractVariant class so you can call them from here. And if the message eventually changes the tests wouldn't have to be changed.